src/App.js

```diff
@@ -0,0 +1,61 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Route, Routes } from 'react-router-dom';
+import { Navigation } from './components/navigation/navigation.component';
+import LoginPage from './pages/login-page/loginPage.component';
+import SignUpPage from './pages/signup-page/signupPage.component';
+import ReservationPage from './pages/reservations-page/reservationsPage.component';
+import NewPropertyPage from './pages/create-property/newProperty-page.component';
+import { Homepage } from './pages/homepage/homepage.component';
+import { PropertyDetailsPage } from './pages/property-details-page/property-details-page.component';
+import { fetchCategoryItemsAsync } from './redux/category/category.actions';
+import { checkUserSessionAsync } from './redux/user/user.actions';
+import { selectCurrentUser } from './redux/user/user.selectors';
+import { ProtectedRoute } from './components/protected-route/protected-route.component';
+import { UserProfilePage } from './pages/user-profile-page/user-profile-page.component';
+import { UserPropertiesPage } from './pages/user-properties-page/user-properties-page.component';
+import { PropertyProfilePage } from './pages/property-profile-page/property-profile-page.components';
+
+export default function App() {
```

Comment:
- [ ] Please, kindly refactor this and all other functional components/functions on your project to ES6 arrow function, so your code complies with the latests standards.

The function keyword has been soft-deprecated all over the software industry.  This means, almost nobody uses it anymore, so much that it's considered non-best practice.

Refactoring  functions to ES6 is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions.

You can do a search in Vscode for the "function" word, and correct as needed I found these:  

![image](https://user-images.githubusercontent.com/57421823/203671861-ce6576a6-1e44-4240-b8d9-bec9f0f51417.png)


It shouldn't take you more than 10 min so don't worry.